api_spec.yaml

```diff
@@ -227,35 +227,33 @@ paths:
         '500':
           $ref: '#/components/responses/500'
 
-  /appid-<appId>/recipe/oauth2/token/revoke:
-    post:
+  /appid-<appId>/<tenantId>/recipe/oauth2/token:
+    delete:
       tags:
         - OAuth2 Recipe
       operationId: revokeOAuth2Token
       description: |
-        Revoke a token or all tokens belonging to a session or to a client (across all tenants).
+        Revoke a token or all tokens belonging to a session or to a client (across all tenants). [Click here](https://app.code2flow.com/V0wfSu1Y1b9U) for flow diagram.
         To provide Single Logout, you should also call the normal session logout endpoint and the token revocation endpoint as well.
       parameters:
         - $ref: '#/components/parameters/oauth2Rid'
         - $ref: '#/components/parameters/api-key'
         - $ref: '#/components/parameters/cdi-version'
-      requestBody:
-        content:
-          application/json:
-            schema:
-              oneOf:
-                - type: object
-                  properties:
-                    accessToken:
-                      type: string
-                - type: object
-                  properties:
-                    sessionHandle:
-                      type: string
-                - type: object
-                  properties:
-                    clientId:
-                      type: string
+        - in: query
+          name: accessToken
```

Comment:
in the flow:
- For session handle and access token, we need to check that they are not expired before using them for deletion